src/main/java/org/folio/circulation/support/StreamToListMapper.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.circulation.support;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class StreamToListMapper {
+  private StreamToListMapper() { }
+
+  public static <T> List<T> toList(Stream<T> stream) {
+    if (stream == null) {
+      return null;
```

Comment:
Why doesn't it return an empty list?

Reply:
Good question. The original intent was to preserve any null that was passed in, just in case any subsequent client specifically cared about the null case.

Upon reflection, I've changed it to return an empty list instead, as clients to these methods should not really be relying on null checks.